### ISSUE TITLE
Can't predict anything when running test.py

### ISSUE THREAD

**Issue by warriors30n11** [YunYang1994/TensorFlow2.0-Examples#89]
After I prepared all the environment required in YOLOv3(Tensorflow2), I successfully run image_demo.py and video_demo.py.They all predict right.
But when I trained the yymnist dataset, I ran train.py without any error, but I ran test.py, there is no any prediction. The terminal show likes this:
![image](https://user-images.githubusercontent.com/50744182/80308397-63448080-8801-11ea-9d2f-798dfb917466.png)

I have changed the class type to yymnist.names and the training dataset.
Here is my config.py:
![image](https://user-images.githubusercontent.com/50744182/80308447-b1598400-8801-11ea-82da-5a5ef582d866.png)
I changed epochs to 1 for saving time and for find the bug.

WHERE IS THE PROBLEM IS?

中文翻譯:
我安裝完yolov3(tensorflow2)所需要的環境後，我跑image_demo.py 和 video_demo.py時都成功顯示而且有box框住物體。
但當我在訓練yymnist dataset的時候，我跑train.py沒有任何問題，但當我跑test.py時，卻無法預測出任何東西，狀況如上面第一張圖。

我確定我在train以及test的時候config內容都沒有錯，config內容如上面第二張圖。
我把epochs改為1單純是為了節省時間方便找出bug。

有人知道我的問題是出在哪嗎

**Comment by warriors30n11** [comment]
Solved!
First, I trained with my dataset, and the epoch is 30. Then I can't predict anything in my dataset.
Therefore I went back to train yymnist to find out where is the problem. And I thought the problem is that the epoch is too short to make any prediction. So I raised the epoch in yymnist(1 to 10), then my yymnist could predict.
But still not working in my dataset by raising epoch.
Suddenly, I thought that the loss value of my dataset  is nan after about 4000 steps. Therefore, I changed the lr_INIT to 1e-4 and warmup_epochs to 4. Then my dataset could predict something.
